### ISSUE TITLE
URL query parameters do not get removed in htu field

### ISSUE THREAD

**Issue by joachimvh** [inrupt/solid-client-authn-js#1842]
### Search terms you've used
htu

### Impacted package

Which packages do you think might be impacted by the bug ?

- [x] solid-client-authn-core

### Bug description
The `htu` field of the access token is generated [here](https://github.com/inrupt/solid-client-authn-js/blob/c3864dc9d978176d682a2c90615d77c29f48a053/packages/core/src/authenticatedFetch/dpopUtils.ts#L61). Actual code of the [removeHashUsernameAndPassword](https://github.com/inrupt/solid-client-authn-js/blob/c3864dc9d978176d682a2c90615d77c29f48a053/packages/core/src/authenticatedFetch/dpopUtils.ts#L33-L39) function is as follows: 
```js
function removeHashUsernameAndPassword(audience: string): string {
  const cleanedAudience = new URL(audience);
  cleanedAudience.hash = "";
  cleanedAudience.username = "";
  cleanedAudience.password = "";
  return cleanedAudience.toString();
}
```

The issue is that the above code does not clear the query parameters of the URL. `http://example.com/?query` remains unchanged after going through that function, but the `htu` parameter is defined as
 > The HTTP URI used for the request, without query and fragment parts

### To Reproduce
See https://github.com/solid/community-server-recipes/issues/3

Or just run the code above with any URL containing a query parameter.

### Expected result
`htu: http://example.com/`

### Actual result
`htu: http://example.com/?query`

**Comment by NSeydoux** [comment]
Thanks for reporting @joachimvh , we'll fix this soon.

**Comment by diegoaraujo** [comment]
Any news?

**Comment by ThisIsMissEm** [comment]
This was released in [1.11.4 and onwards](https://github.com/inrupt/solid-client-authn-js/blob/main/CHANGELOG.md#1114---2022-02-11), sorry we didn't follow up properly on this. We've also just done a fix for the `iss` parameter in #2103, which should be released soon-ish.

Closing as this is done.
